test/e2e/quick_start_test.go

```diff
@@ -278,4 +280,41 @@ func TestQuickStart(t *testing.T) {
 	if err != nil {
 		t.Fatalf("failed to ensure guest nodes became ready: %s", err)
 	}
+
+	clusterOperators := &configv1.ClusterOperatorList{}
+	err = wait.Poll(10*time.Second, 10*time.Minute, func() (done bool, err error) {
```

Comment:
After rebasing from https://github.com/openshift/hypershift/pull/77 you can refactor this to set up a local context with the timeout you want based on the parent context in the test setup so that timeout propagation will work correctly